### PR TITLE
Issue 5 shortcuts

### PR DESCRIPTION
**Pull request by tfrancart**
@florenceclavaud you can validate this PR that adds the shortcut properties. Thanks

### CODE REVIEW

File: ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico.xslt [hunk 1]
```diff
@@ -688,6 +694,9 @@
             <xsl:apply-templates />
          	</rico:TypeRelation>
       </rico:thingIsTargetOfTypeRelation>
+	  <rico:hasOrHadCategory>
```
Comment (florenceclavaud) [inline ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico.xslt:697]: IMHO it should rather be `rico:hasOrHadCorporateBodyType` here (we have a TypeRelation that has object a CorporateBodyType; and when the legalStatus element has not date or descriptive note, it is converted into a rico:hasCorporateBodyType

**Comment by tfrancart** [comment]
OK, this is fixed

File: ricoconverter/ricoconverter-convert/src/test/resources/eac2rico/_22occupation/expected.xml [hunk 2]
```diff
@@ -24,19 +24,20 @@
...
-               <rico:Activity>
+               <rico:Activity rdf:nodeID="_Activity-a">
                   <rico:activityIsSourceOfPerformanceRelation rdf:resource="performanceRelation/005205-d699msj02xm-19qbb7wdtto3j"/>
                   <!-- note that the following assertion is possible only with RiC-O v0.2, where OccupationType has become a subclass of ActivityType-->
                   <rico:hasActivityType rdf:resource="occupationType/FRAN_RI_010-d699msj02xm-19qbb7wdtto3j"/>
                </rico:Activity>
             </rico:performanceRelationHasSource>
          </rico:PerformanceRelation>
       </rico:agentIsTargetOfPerformanceRelation>
+      <rico:performsOrPerformed rdf:nodeID="_Activity-a" />
```
Comment (florenceclavaud) [inline ricoconverter/ricoconverter-convert/src/test/resources/eac2rico/_22occupation/expected.xml:35]: Hi @tfrancart, here IMHO the following line can be added (as the entity is a Person that performs an Activity that has type an OccupationType):
`<rico:hasOrHadOccupationOfType rdf:resource="occupationType/FRAN_RI_010-d699msj02xm-19qbb7wdtto3j"/>`
Same below line 52

File: ricoconverter/ricoconverter-convert/src/test/resources/eac2rico/_22occupation/expected.xml [hunk 3]
```diff
@@ -48,5 +49,6 @@
             </rico:descriptiveNote>
          </rico:PerformanceRelation>
       </rico:agentIsTargetOfPerformanceRelation>
+      <rico:performsOrPerformed rdf:nodeID="_Activity-b" />
```
Comment (florenceclavaud) [inline ricoconverter/ricoconverter-convert/src/test/resources/eac2rico/_22occupation/expected.xml:52]: Same as above: the following line can be added:
`<rico:hasOrHadOccupationOfType rdf:resource="occupationType/FRAN_RI_010-FRAN_RI_010-d3nyub73ic-1g5es55lb2lpf"/>`

File: ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico.xslt [hunk 4]
```diff
@@ -654,6 +659,7 @@
 				<xsl:apply-templates />
 			</rico:PerformanceRelation>
 		</rico:agentIsTargetOfPerformanceRelation>
```
Comment (florenceclavaud) [inline ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico.xslt:661]: I suggest we add the following line here:
`<rico:hasOrHadOccupationOfType>
   <xsl:call-template name="rdf-resource"><xsl:with-param name="uri" select="eac2rico:URI-OccupationType(eac:term/@vocabularySource)" /></xsl:call-template>
</rico:hasOrHadOccupationOfType>`

**Review by florenceclavaud — COMMENTED** [review]
Everything is OK. I have added 3 specific comments to 2 files (the eac2rico XSLT and the expected file in the unit test) as concerns the way the EAC-CPF occupation element is processed: IMHO a line should be added to say that the person hasOrHadOccupationOfType the OccupationType ; this is a kind of shortcut.

**Comment by tfrancart** [comment]
Regarding your comments about rico:hasOrHadOccupationOfType : this is currently on branch issue-17-relationAgentActivity where you introduced this change, so we will tackle it on this branch.

**Pull request merged**